Set-Game/Set-Game.xcodeproj/project.pbxproj

```diff
@@ -15,6 +15,11 @@
 		C5DC573522DDB65500AC9D8B /* Card.swift in Sources */ = {isa = PBXBuildFile; fileRef = C5DC573422DDB65500AC9D8B /* Card.swift */; };
 		C5DC573722DDBD2F00AC9D8B /* Deck.swift in Sources */ = {isa = PBXBuildFile; fileRef = C5DC573622DDBD2F00AC9D8B /* Deck.swift */; };
 		C5DC573922DDDBF800AC9D8B /* Game.swift in Sources */ = {isa = PBXBuildFile; fileRef = C5DC573822DDDBF800AC9D8B /* Game.swift */; };
+		C5FDEEC422EF0677005594C5 /* Grid.swift in Sources */ = {isa = PBXBuildFile; fileRef = C5FDEEC322EF0677005594C5 /* Grid.swift */; };
```

Comment:
Please try to make the app work on different screen sizes i.e. try to run it on iPad and see what happens.
Hint: it is because in viewDidLoad you don'y have geometry yet i.e. the view was just loaded into memory only once it was added to the view hierarchy the views geometry is set. Check https://www.appcoda.com/view-controller-lifecycle/

Reply:
Make sure the cards are displayed correctly in landscape too